.github/.dependabot.yaml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
it doesnt' work with yarn 2+

Reply:
for dependa bot, I followed instructions from the handbook - https://engineering-handbook.onrender.com/development/tech-stack/typescript#dependabot

Reply:
thats for yarn 1. https://github.com/ChainSafe/engineering-handbook/pull/28/files#diff-65d1500c609e6708a8709aedda33ce9f13585733e71bd56188bd9f9dda8eefe9R125